src/DynamoCoreWpf/UI/LibraryTreeTemplateSelector.cs

```diff
@@ -43,7 +44,12 @@ public override DataTemplate SelectTemplate(object item, DependencyObject contai
                 return MemberGroupsTemplate;
 
```

Comment:
```csharp
// "Top Result" is no longer a standalone panel on the library view. A SearchCategory 
// is created based off the first item in search results, and inserted into the results 
// just like any other SearchCategory objects. The only difference is, "IsTopCategory"
// property will be set to "true". This is where the right template is selected so that 
// top result item appears to look different from other categories in results.
//
```